server/libs/infra/src/repositories/media.repository.ts

```diff
@@ -22,11 +28,31 @@ export class MediaRepository implements IMediaRepository {
         return;
 
       case 'jpeg':
-        await sharp(input, { failOnError: false })
-          .resize(options.size, options.size, { fit: 'outside', withoutEnlargement: true })
-          .jpeg()
-          .rotate()
-          .toFile(output);
+        if (!options.raw) {
+          await sharp(input, { failOnError: false })
+            .resize(options.size, options.size, { fit: 'outside', withoutEnlargement: true })
+            .jpeg()
+            .rotate()
+            .toFile(output);
+        } else {
+          try {
+            await stat(output);
+            await unlink(output);
```

Comment:
What's the purpose of this? Why is it okay for it to fail?

Reply:
The dark table cli don't ever overwrite a file that exists and trying to stat a file that doesn't exist throws an error.

Reply:
I think I see, thanks. In which case, something like this might be more suitable?

```ts
try {
  await unlink(output);
} catch(e) {
  if (e && e.code !== 'ENOENT') throw e;
}
```

What do you think?

Reply:
Possibly, yeah